src/components/ProjectSettingsTabs/CreateProjectModal.tsx

```diff
@@ -109,13 +120,13 @@ export const CreateProjectModal: React.FC<Props> = ({ owner, repo }) => {
     refetchQueries: ["ProjectSettings", "RepoSettings"],
   });
 
-  if (!isAdmin) {
+  if (!canCreateProject) {
```

Comment:
can we fallback to something?

Reply:
Can you please elaborate a bit more? What kind of fallback? If someone doesn't have permissions, we don't want them to see the button. Do you mean like an empty div or something?

Reply:
Sorry i missed the context of this change i didn't realize this was for the button that toggles the modal.